crates/sui-transactional-test-runner/src/args.rs

```diff
@@ -228,6 +228,18 @@ pub struct SetRandomStateCommand {
     pub randomness_initial_version: u64,
 }
 
+#[derive(Debug, clap::Parser)]
+pub struct AuthenticatorStateUpdateCommand {
+    #[clap(long = "round")]
+    pub round: u64,
+    /// Comma-separated list of JWK issuers (e.g., "google.com,microsoft.com").
+    /// Key IDs will be automatically generated as "key1", "key2", etc.
+    #[clap(long = "jwk-iss", value_delimiter = ',')]
```

Comment:
nit (optional): We tend to use `arg_action = ArgAction::Append` instead (so you provide multiple keys by supplying the flag multiple times) -- this avoids having to introduce another delimiter character.